src/main/java/uk/gov/pay/connector/token/TokenService.java

```diff
@@ -19,16 +20,16 @@ public TokenService(TokenDao tokenDao) {
         this.tokenDao = tokenDao;
     }
 
-    public void validateTokenForMotoApi(String oneTimeToken) {
-        tokenDao.findByTokenId(oneTimeToken)
+    public ChargeEntity getChargeForTokenForMotoApi(String oneTimeToken) {
```

Comment:
Shall we expliciltly say it also validates one time token ?
 
```suggestion
    public ChargeEntity validateAndGetChargeForTokenForMotoApi(String oneTimeToken) {
```

Reply:
Yeh ok